src/main/java/com/traxel/lumbermill/event/TableView.java

```diff
@@ -162,14 +162,6 @@ public Component prepareRenderer(final TableCellRenderer renderer, final int row
             } catch (final Exception e) {
                 // LOG.error("error while preparing renderer", e);
             }
-            if (!((renderer instanceof SeverityCellView)
-                            || this.getSelectionModel().isSelectedIndex(rowIndex))) {
-                if (event instanceof XMLEvent) {
-                    jc.setBackground(new Color(236, 233, 216));
```

Comment:
Just come back to this feature yesterday (didn't have much time).

The fact is if I do not remove the entire if statement, I cannot see manually loaded log4j events with my dark theme.

Which look and feel do you use ? I ran some tests and I didn't manage to reproduce the look of your screenshot.

Thanks